src/dappsExplorer/TabDiscover.test.tsx

```diff
@@ -365,6 +361,57 @@ describe('TabDiscover', () => {
             },
           },
         },
+        positions: {
+          positions: [
```

Comment:
nit: can we use `mockEarnPositions` from test/values and update the balance as required?